src/components/menus/directory-tree-contextual-menu.tsx

```diff
@@ -96,7 +96,7 @@ export default function DirectoryTreeContextualMenu(props: Readonly<DirectoryTre
 
     const [renameCB, renameState] = useDeferredFetch(
         renameElement,
-        () => handleCloseDialog(null, null),
+        () => handleCloseDialog(),
```

Comment:
Why not just pass handleCloseDialog, it's already a callback

Reply:
DONE in https://github.com/gridsuite/gridexplore-app/pull/540/commits/6c117350141ea6709558c2e41df7797be49220b2